rv-monitor/src/main/java/com/runtimeverification/rvmonitor/java/rvj/output/combinedoutputcode/CombinedOutput.java

```diff
@@ -215,6 +219,16 @@ public String toString() {
 
         ret += this.indexingTreeManager.decl();
 
+        if (Main.controlAPI) {
+            ret += "public static final void enable() {\n";
```

Comment:
FWIW, this seems in line with the reminder of the function, so you can leave it as-is, but the standard way of doing string concatenation is to use `StringBuilder`, otherwise the complexity is O(m*n) where m is the number of strings and n is the total length, or something like that.